src/core/io/writer/h5md_core.hpp

```diff
@@ -27,17 +27,15 @@
 #include "cells.hpp"
 #include "global.hpp"
 
+#include "communication.hpp"
```

Comment:
Please clean up the header. I think the core headers are only needed in the implementation..